src/rulesets/oas2/index.ts

```diff
@@ -1,6 +1,20 @@
+import { FunctionCollection } from '../../types';
 import { readRuleset } from '../reader';
 
-export { commonOasFunctions as oas2Functions } from '../oas';
+import { commonOasFunctions } from '../oas';
+
+export { commonOas2Functions as oas2Functions };
+
+const commonOas2Functions = (): FunctionCollection => {
+  console.warn('This is deprecated. Use loadRuleset method instead');
+
+  const oas2Functions = {
```

Comment:
Just return this directly?

Reply:
Fixed

Reply:
Resolving this discussion, as the topic seem closed. Feel free to unresolve it would I have missed anything.